src/Context/FixtureContext.php

```diff
@@ -989,20 +989,8 @@ protected function selectInTheHtmlField(string $select, string $field)
         $inputField->getParent()->find('css', 'iframe')->click();
         $inputFieldId = $inputField->getAttribute('id');
         $js = <<<JS
-        var editor = jQuery('#$inputFieldId').entwine('ss').getEditor(),
-            doc = editor.getInstance().getDoc(),
-            sel = doc.getSelection(),
-            rng = new Range(),
-            matched = false;
-
-        jQuery(doc).find("$select").each(function() {
-            if(!matched) {
-                rng.selectNode(this);
-                sel.removeAllRanges();
-                sel.addRange(rng);
-                matched = true;
-            }
-        });
+        const editor = jQuery('#$inputFieldId').entwine('ss').getEditor();
+        editor.selectByCssSelector("$select");
```

Comment:
The TinyMCE-specific logic here was moved into the tinymce module. The `selectByCssSelector()` function is in admin, and ultimately ends up calling editor-specific functionality.